lib/power_action_utils.pm

```diff
@@ -203,17 +240,17 @@ sub handle_livecd_reboot_failure {
 
 =head2 power_action
 
-    power_action($action [,observe => $observe] [,keepconsole => $keepconsole] [,textmode => $textmode]);
+ power_action($action [,observe => $observe] [,keepconsole => $keepconsole] [,textmode => $textmode]);
+
+Executes the selected power action (e.g. poweroff, reboot). 
+
+If C<$observe> is set the function expects that the specified C<$action> was already executed by
+another actor and the function justs makes sure the system shuts down, restart etc. properly. 
 
-Executes the selected power action (e.g. poweroff, reboot). If C<$observe> is
-set the function expects that the specified C<$action> was already executed by
-another actor and the function justs makes sure the system shuts down, restart
-etc. properly. C<$keepconsole> prevents a console change, which we do by
-default to make sure that a system with a GUI desktop which was in text
-console at the time of C<power_action> call, is switched to the expected
-console, that is 'root-console' for textmode, 'x11' otherwise. The actual
-execution happens in a shell for textmode or with GUI commands otherwise
-unless explicitly overridden by setting C<$textmode> to either 0 or 1.
+C<$keepconsole> prevents a console change, which we do by default to make sure that a system with a GUI 
```

Comment:
I would re-write the documentation for this function to be more clear and to not use such a long sentences like the one I've highlighted: `C<$keepconsole> prevents a console change,...`, as currently it requires a lot of concentration to understand what is described here (at least for me). 

But it is a matter of taste and if you'd like it, just keep it:)

Reply:
I think let's keep this for now. We can check this later when we have more time